packages/eslint-config-base/index.js

```diff
@@ -24,24 +20,13 @@ module.exports = {
         ignoreRegExpLiterals: true,
       },
     ],
+    'no-console': ['error', { allow: ['warn', 'error'] }],
+    'no-param-reassign': ['error', { props: false }],
     'new-parens': 'off',
+    'no-plusplus': ['error', { allowForLoopAfterthoughts: true }],
+    'import/order': ['error', { 'newlines-between': 'always' }],
```

Comment:
What about to have 2 empty lines after the last group. It allows us to easily distinguish imports and the rest of the file. https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md#example-options-usage


Didn't you miss group parameter for 'import/order'? Now we will have 5 groups instead of three. Or is it intended?

Reply:
1. We never import build-ins.
2. I would like to distinguish parents/siblings but I would like to keep it together. Do you know how we can avoid new line there?
3. 2 lines after last group conflict with prettier, so we basically cannot.

Reply:
1. 🆗 
2. I thought about the same. I am afraid it's not possible.
3. 😨 🆗